From 1f2cdfc510e2fb9eedefd23bf78404ff0b204cdb Mon Sep 17 00:00:00 2001
From: Tomas Glozar <tglozar@gmail.com>
Date: Fri, 20 Mar 2026 20:00:52 +0100
Subject: [PATCH 2/8] ia64: Add Linux Security Module syscalls

lsm_get_self_attr(), lsm_set_self_attr(), and lsm_list_modules() were
added after ia64 was removed from the kernel.

Fixes: 5f42375904b0 ("LSM: wireup Linux Security Module syscalls")
Signed-off-by: Tomas Glozar <tglozar@gmail.com>
---
 arch/ia64/kernel/syscalls/syscall.tbl | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/ia64/kernel/syscalls/syscall.tbl b/arch/ia64/kernel/syscalls/syscall.tbl
index 9064b3c8128d..636854b52b8a 100644
--- a/arch/ia64/kernel/syscalls/syscall.tbl
+++ b/arch/ia64/kernel/syscalls/syscall.tbl
@@ -379,3 +379,6 @@
 456	common	futex_requeue			sys_futex_requeue
 457	common	statmount			sys_statmount
 458	common	listmount			sys_listmount
+459	common	lsm_get_self_attr		sys_lsm_get_self_attr
+460	common	lsm_set_self_attr		sys_lsm_set_self_attr
+461	common	lsm_list_modules		sys_lsm_list_modules
-- 
2.52.0

